docs/editor/accessibility.md

```diff
@@ -9,59 +9,59 @@ MetaDescription: Visual Studio Code user accessibility features.  Learn here abo
+* **View** > **Appearance** > **Zoom In** (`kb(workbench.action.zoomIn)`) - increase the zoom level.
+* **View** > **Appearance** > **Zoom Out** (`kb(workbench.action.zoomOut)`) - decrease the zoom level.
+* **View** > **Appearance** > **Reset Zoom** (`kb(workbench.action.zoomReset)`) - reset the zoom level to 0.
 
->**Note**: If you are using a magnifier make sure to hold the `alt` key while viewing the hover to allow the mouse to move over the hover.
+>**Note**: If you're using a magnifier, hold down the `alt` key while viewing the hover to move the cursor over the hover.
 
 ![Zoomed in editor](images/accessibility/zoomed-in.png)
 
-### Persisted Zoom Level
+### Persistent zoom level
 
-When you adjust the zoom level with the **View** > **Zoom In / Out** commands, the zoom level is persisted in the `window.zoomLevel` [setting](/docs/getstarted/settings.md). The default value is 0 and each increment/decrement changes the zoom level by 20%.
+When you adjust the zoom level with the **View** > **Zoom In** or  **Zoom Out** commands, the zoom level is persisted in the [`window.zoomLevel` setting](/docs/getstarted/settings.md). The default value is 0, and each increment or decrement changes the zoom level by 20 percent.
 
 ## High Contrast theme
 
-We support a High Contrast color theme on all platforms.  Use **File** > **Preferences** > **Theme** > **Color Theme** (`kb(workbench.action.selectTheme)`) to display the **Select Color Theme** dropdown and select the **High Contrast** theme.
+VS Code supports a High Contrast color theme on all platforms.  Go to **File** > **Preferences** > **Theme** > **Color Theme** (`kb(workbench.action.selectTheme)`) to display the **Select Color Theme** dropdown and select the **High Contrast** theme.
 
-![High Contrast Theme](images/accessibility/high-contrast.png)
+![High Contrast theme](images/accessibility/high-contrast.png)
 
 ## Color vision accessibility
 
-You can search for extensions in **Visual Studio Marketplace** that are compatible for color vision deficiency. Use the Extensions view `kb(workbench.view.extensions)` and search for "colorblind" to populate relevant options.
+You can search for extensions in **Visual Studio Marketplace** that are compatible with color vision deficiency. Use the Extensions view `kb(workbench.view.extensions)` and search for "colorblind" to show relevant options.
 
 ![Visual Studio Marketplace in VS Code UI](images/accessibility/accessibility-extension-marketplace.png)
 
-Once you have installed a color theme from the Marketplace, you can change the [color theme](/docs/getstarted/themes.md) with **File** > **Preferences** > **Theme** > **Color Theme** `kb(workbench.action.selectTheme)`.
+Once you've installed a color theme from the marketplace, you can [change the color theme](/docs/getstarted/themes.md) with **File** > **Preferences** > **Theme** > **Color Theme** `kb(workbench.action.selectTheme)`.
```

Comment:
Marketplace is capitalized

Reply:
This one is thorny—or other Microsoft purchasing experiences, we aren't supposed to refer to them as JUST "Marketplace," it always needs to be "Azure Marketplace" or "Xbox Marketplace" or whatever. Recognizing that "Visual Studio Marketplace" is a longish phrase, I was trying to split the difference. The preferred generic noun would be "store," but since we haven't referred to it as a "store," it wouldn't hurt the sentence to just trim those few words: "Once you've installed a color theme, you can ..."